api/src/handlers/blocks_api.rs

```diff
@@ -144,16 +149,18 @@ impl Handler for BlockHandler {
 		if let Some(param) = req.uri().query() {
 			if param == "compact" {
 				result_to_response(self.get_compact_block(&h))
+			} else if param == "include_proof" {
+				result_to_response(self.get_block(&h, true, true))
 			} else if param == "no_merkle_proof" {
-				result_to_response(self.get_block(&h, false))
+				result_to_response(self.get_block(&h, false, false))
```

Comment:
These permutations are getting a bit complex with multiple combinations of true/false for the two proofs.
Not for this PR but we should probably think about passing a struct around - build it based on query params and pass it to `build_block()`.

Reply:
Yes, makes sense